File: atividade07.c
#include <stdio.h>

int num;

int main()
{
    int contador = 0;

    for(int i = 0; i<=10; i++)
    {
        printf("Informe um Número: ");
        scanf("%d", &num);
        printf("\n");

        if(num >= 30 && num <=90)
        {
            contador++;
        }
    }

    printf("Quantidade de numeros entre 30 e 90: %d",contador);
return 0;
}

File: atividade_py/atividade07.py
contador = 0;

for i in range(1,11):
    print(f"Informe o {i}º número: ",end="");
    num = int(input());
    if num >= 30 and num <= 90:
        contador += 1;

print(f"Quantidade de numeros entre 30 e 90: {contador}");
